src/controllers/class.controller.js

```diff
@@ -27,14 +27,32 @@ const deleteClass = catchAsync(async (req, res) => {
 });
 
 const getAllClasses = catchAsync(async (req, res) => {
-  const _classes = await classService.getAllClasses(req.user._id, req.user._id);
+  const _classes = await classService.getAllClasses(req.user._id);
```

Comment:
why did you undo this change? we need the teacher id to just return the classes of the teacher

Reply:
ahh, i think i removed the content in master branch when i merged your master branch and my set-up branch. I change it back

Reply:
Oh i am sorry you are right